### PR TITLE
jormungandr: Add text color in display information

### PR DESCRIPTION
**Pull request by kadhikari**

### CODE REVIEW

File: source/jormungandr/tests/departure_board_tests.py [hunk 1]
```diff
@@ -416,6 +416,25 @@ def test_line_closed(self):
         assert ss2_d['additional_informations'] == 'no_active_circulation_this_day'
         assert ss3_d['additional_informations'] == 'partial_terminus'
 
+    def test_display_informations_in_routes_schedule(self):
+        """
+        departure board for a given date
```
Comment (antoine-de) [inline source/jormungandr/tests/departure_board_tests.py]: unsynchronized comment

**Comment by antoine-de** [comment]
apart from my comment it's OK for me

**Comment by kadhikari** [comment]
Comment modified

**Comment by aromatix** [comment]
Jenkins, test this please

**Pull request merged**